### PR TITLE
Add schemas

### PR DESCRIPTION
**Pull request by darkBasic666**
Add schemas structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Now, to start your app run ```npm start``` in the rootpath of the project. Then
 ## Development
 
 #### Environments
-By default, the environment will be **development**, but you can easily change it using the **NODE_ENV** environmental variable.
+By default, the environment will be **development**, but you can easily change it using the **NODE_ENV** environmental variable. IIn this files you can configure a DB connection and the schema it should use (the default ispublic).
```
Comment (gonziesc) [inline README.md]: Typo `IIn`

Comment (gonziesc) [reply README.md]: Also `ispublic`

Comment (gonziesc) [reply README.md]: I'll write something like `In this files you can set your DB connection and the schema that should use (default: public)`. Also, the files should be linked here.

File: app/models/utils.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const config = require('./../../config/');
+
+exports.buildDBOptions = () => {
```
Comment (gonziesc) [inline app/models/utils.js]: I'd call it options

File: app/models/book.js [hunk 3]
```diff
@@ -1,4 +1,7 @@
-const Sequelize = require('sequelize');
+const Sequelize = require('sequelize'),
+  utils = require('./utils');
```
Comment (gonziesc) [inline app/models/book.js]: `options = require('./utils').options()`

File: config/development.js [hunk 4]
```diff
@@ -1,4 +1,27 @@
 exports.config = {
   environment: 'development',
+  common: {
```
Comment (gonziesc) [inline config/development.js]: This should not be here

File: config/production.js [hunk 5]
```diff
@@ -1,4 +1,28 @@
 exports.config = {
   environment: 'production',
+  common: {
```
Comment (gonziesc) [inline config/production.js]: same

File: config/staging.js [hunk 6]
```diff
@@ -1,4 +1,27 @@
 exports.config = {
   environment: 'staging',
+  common: {
```
Comment (gonziesc) [inline config/staging.js]: same

File: app/models/we-users-4.sql [hunk 7]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO we_wolox_schema."user"("firstName","lastName","laborHours","firstDayDate","birthday","career","university","email","created_at","updated_at") VALUES ('Veronica','Insaurralde',20,'2017-09-18T00:00:00-03:00','1992-03-15T00:00:00-03:00','Profesorado en Computacion','UBA','veronica.insaurralde@wolox.com.ar',current_timestamp,current_timestamp);
```
Comment (gonziesc) [inline app/models/we-users-4.sql]: Delete this

File: app/orm.js [hunk 8]
```diff
@@ -8,7 +8,6 @@ exports.DB_URL =
   `postgres://${config.common.database.username}:${config.common.database.password}@${
     config.common.database.host
   }:${config.common.database.port}/${config.common.database.name}`; // eslint-disable-line max-len
```
Comment (gonziesc) [inline app/orm.js]: Reinsert tab

**Comment by gonziesc** [comment]
Don't remove package lock, just do not upload yours if you did not inserted a dependency

**Comment by gonziesc** [comment]
i think we will close this...

**Comment by mpicollo-wolox** [comment]
I think we should @gonziesc

**Pull request closed without merging**